packages/application-shell/src/test-utils/test-utils.tsx

```diff
@@ -429,12 +408,7 @@ function renderApp<AdditionalEnvironmentProperties = {}>(
         apolloClient={apolloClient}
         mocks={mocks}
       >
-        <ConfigureFlopFlip
-          adapter={adapter}
-          defaultFlags={flags}
-          adapterArgs={defaultFlopflipAdapterArgs}
-          shouldDeferAdapterConfiguration={!hasFlags}
-        >
+        <TestProviderFlopFlip flags={flags}>
```

Comment:
Hmm wait, do we still need the change in `SetupFlopFlip` then? 🤔

Reply:
Yes, otherwise our tests here fail of app-shell if this is removed. I think removing it, if possible, could also come after.